dashboard/config/locales/en.yml

```diff
@@ -33,6 +33,7 @@ en:
     attributes:
       user:
         name: 'Display Name'
+        name_example: 'Display Name (e.g. Cool Coder or Jane S.)'
```

Comment:
Can you add a `<br>` tag between the field name and help text?

![image](https://cloud.githubusercontent.com/assets/1920530/22913362/209a18b0-f21f-11e6-9020-7cb816ff69e6.png)